src/components/NcRichText/NcReferenceWidget.vue

```diff
@@ -317,17 +317,9 @@ export default {
 	position: relative;
 	.toggle-interactive--button {
 		position: absolute;
-		top: 50%;
+		bottom: var(--default-grid-baseline);
+		right: var(--default-grid-baseline);
```

Comment:
IMO, it looks better, but it can overflow the content

Reply:
We can consider:
- make a button semi-transparent when not hovered;
- make it icon button (and put text in title/aria-label

Don't see it moved outside of widget area